openprescribing/frontend/tests/test_api_spending.py

```diff
@@ -34,48 +34,50 @@ def _create_prescribing_tables():
 
 
 class TestAPISpendingViewsTariff(ApiTestBase):
+    fixtures = ApiTestBase.fixtures + ['dmd-subset-1', 'tariff']
```

Comment:
No sure where this was first defined, but could we give `dmd-subset-1` a more descriptive name to indicate what it's suitable for if someone wants to reuse it? (Or if we think that's not a good idea, a name that suggests which test(s) it is bound to?)

Reply:
Have renamed it to `dmd-subset`.